background/rust.tex

```diff
@@ -40,6 +40,10 @@ \subsubsection{The Rust Standard Library}
 This basic fact renders the \gls{rsl} unusable in an embedded system as these dependencies will not be met.
 It is possible to opt-out of using the library.
 
+It is important to note that there is a distinction between the actual software component called {\std}, and the library we refer to as \gls{rsl}.
```

Comment:
Hmm, synes ikke det er helt god korrigering mellom dette avsnittet og det over.

Over står det bl.a. at RSL er en "...facade, meaning that RSL provides a thin wrapper which reexports..."